app/models/cohort.rb

```diff
@@ -12,26 +12,30 @@ class Cohort < ApplicationRecord
   scope :current, -> { where('open_date <= ? AND close_date >= ?', Date.today, Date.today) }
   scope :closed, -> { where('open_date > ? OR close_date < ?', Date.today, Date.today) }
 
+  def order_by_id_and_state
```

Comment:
I think I'd name this method something like `applications_ordered_by_blahblah` to make it clear it deals with apps as opposed to sorting cohorts

Reply:
👍